### ISSUE TITLE
Can not inspect elements for our APP using inspect.exe

### ISSUE THREAD

**Issue by njumazhen** [microsoft/WinAppDriver#1146]
Hello, I want to confirm, if we want to use WinAppDriver to auto test our App on windows, what special work we need to do before the it can be auto test using WinAppDriver? As now, the element can not be found using the inspect.exe. What need to inplemented to support auto test? I found an answer that mentionded the App need to support/work with UI Automationin in another similar issue? https://docs.microsoft.com/zh-cn/windows/win32/winauto/uiauto-clientsideprovider
If so, we must change the exsiting code?
Any sugggestion is apprecaite. Thanks!
![image](https://user-images.githubusercontent.com/47347911/81409245-5add3580-9171-11ea-869b-99cf546c5a6c.png)

**Comment by anunay1** [comment]
Yes u need to change the existing code for the inspect to find the element.

**Comment by njumazhen** [comment]
> xisting code for the inspect to find the element.

@anunay1 Thank you for your reply. I have another quesstion here: Will WinAppDriver support WebView(hybrid app that embedded webview in the native app) test? If yes, when? Thanks!

**Comment by anunay1** [comment]
I have no idea and the support of hybrid apps.
